opal/mca/btl/tcp/btl_tcp_endpoint.c

```diff
@@ -406,12 +397,30 @@ mca_btl_tcp_endpoint_send_blocking(mca_btl_base_endpoint_t* btl_endpoint,
 
 static int mca_btl_tcp_endpoint_send_connect_ack(mca_btl_base_endpoint_t* btl_endpoint)
 {
-    /* send process identifier to remote endpoint */
+
     opal_process_name_t guid = opal_proc_local_get()->proc_name;
 
+    int len = (int) strlen(mca_btl_tcp_magic_id_string);
+
+    /* send magic string to the remote endpoint, identifying me as a
+       fellow Open MPI TCP BTL */
+    if (mca_btl_tcp_endpoint_send_blocking(btl_endpoint,
```

Comment:
why incurring the cost of 2 blocking sends when it could have been done with a single one ?